plugins-compat-tester/src/main/java/org/jenkins/tools/test/maven/ExternalMavenRunner.java

```diff
@@ -26,6 +26,7 @@ public ExternalMavenRunner(File mvn) {
     public void run(Config config, File baseDirectory, File buildLogFile, String... goals) throws PomExecutionException {
         List<String> cmd = new ArrayList<String>();
         cmd.add(mvn.getAbsolutePath());
+        cmd.add("--show-version");
```

Comment:
Not strictly needed, but I found this complementarily helpful to get the corresponding output at the beginning of the mvn calls:

![image](https://user-images.githubusercontent.com/223853/50344602-f42f8b80-052b-11e9-8ebf-b7926f07cdf6.png)